oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I dont think it is a good idea to have a private key in the recipe. 
I understand it is just for debugging but the term "private" do have a meaning. =)

Reply:
true :) its just a key pair to make the bundle ready out of the box, only for dev env, with a warning very explicit on bundle install to not use them in prod.

I see 2 options:

Reply:
I think the latter is better. 

The former will make the key unusable.

Reply:
yes indeed, fixed :)